website/static/js/notificationsTreebeard.js

```diff
@@ -173,9 +173,18 @@ function ProjectNotifications(data) {
                                     subscribe(item, ev.target.value);
                                 }},
                                 [
-                                    item.data.event.title !== 'comment_replies' ? m('option', {value: 'none', selected : item.data.event.notificationType === 'none' ? 'selected': ''}, 'Never') : null,
-                                    m('option', {value: 'email_transactional', selected : item.data.event.notificationType === 'email_transactional' ? 'selected': ''}, 'Instantly'),
-                                    m('option', {value: 'email_digest', selected : item.data.event.notificationType === 'email_digest' ? 'selected': ''}, 'Daily')
+                                    item.data.event.title !== 'comment_replies' && !~item.data.event.title.indexOf('mentions') ?
+                                        m('option', {value: 'none', selected : item.data.event.notificationType === 'none' ? 'selected': ''}, 'Never') :
+                                        null,
+                                    ~item.data.event.title.indexOf('mentions') ?
```

Comment:
Gosh, really? That's not the behavior I seem to find in the console. It looks like it adds 1 and flips the sign.

![screen shot 2016-06-07 at 3 01 34 pm](https://cloud.githubusercontent.com/assets/134455/15871084/c55af088-2cc0-11e6-8523-c2a6088c1145.png)